README.md

```diff
@@ -1,2 +1,12 @@
 Terrific traps 
 
+It contains a list of items that are there on the internet that you can buy but they do not have much use. 
+
+For Example:
+1) Bacon Bandages like bacon is for eating not bandaging.
```

Comment:
The added examples in the README.md file are humorous and engaging. However, it would be better to provide a brief explanation of each example to make it more informative for the readers. [medium]